lib/App/Sqitch/Command/check.pm

```diff
@@ -0,0 +1,208 @@
+package App::Sqitch::Command::check;
+
+use 5.010;
+use strict;
+use warnings;
+use utf8;
+use Locale::TextDomain qw(App-Sqitch);
+use App::Sqitch::X qw(hurl);
+use Moo;
+use App::Sqitch::Types qw(Str Target Engine Change);
+use Try::Tiny;
+use namespace::autoclean;
+
+extends 'App::Sqitch::Command';
+with 'App::Sqitch::Role::ContextCommand';
+with 'App::Sqitch::Role::ConnectingCommand';
+
+# VERSION
+
+has target_name => (
+    is  => 'ro',
+    isa => Str,
+);
+
+has target => (
+    is      => 'rw',
+    isa     => Target,
+    handles => [qw(engine plan plan_file)],
+);
+
+has date_format => (
```

Comment:
Don't think you need this attribute. Should probably model attributes based on the `verify` command.